src/dsl/static.jl

```diff
@@ -101,14 +101,26 @@ function parse_trace_expr!(stmts, bindings, fn, args, addr)
     return name
 end
 
+function set_module_for_global_constants(expr, bindings, __module__)
+    expr = MacroTools.postwalk(expr) do e
+        if MacroTools.@capture(e, var_Symbol) && !haskey(bindings, var) && var != :end
+            :($__module__.$var)
```

Comment:
If I'm understanding, the reason to have `:($__module__.$var)` instead of `esc(var)` is to make sure that `var` is bound in the proper module--is this right?

Reply:
I think `esc` might also work, but the semantics of `esc` have often confused me, and I don't mind that we're doing this more specifically!